index.html

```diff
@@ -32,25 +32,22 @@ <h2>My Three Favorite ball games Basketball, football and tennis</h2>
             </ol>
         </section>
         <section>
-            <span style="background: rgb(255, 207, 102);display: block; padding: 1px;padding-left: 15px">
             <h2 style="color: #b400ff;">Haiku</h2>
             <p> An old silent pond...<br> A frog jumps into the pond,<br> splash! Silence again.</p>
             <p>Autumn moonlight-<br> a worm digs silently<br> into the chestnut.</p>
-            <p>In the twilight rain<br> these brilliant-hued hibiscus -<br> A lovely sunset.<br></p>
+            <p><span style="background: rgb(255, 207, 102);">In the twilight rain<br> these brilliant-hued hibiscus -<br> A lovely sunset.</span><br>
```

Comment:
again, `<span>` is an inline element so must go inside of `<p>` which is a block element